frontend/reducers/builds.js

```diff
@@ -0,0 +1,30 @@
+import {
+  buildsFetchStartedType as BUILDS_FETCH_STARTED,
+  buildsReceivedType as BUILDS_RECEIVED,
+  buildRestartedType as BUILD_RESTARTED,
+} from "../actions/actionCreators/buildActions";
+
+const initialState = {
+  isLoading: false,
+}
+
+export default function buildLogs(state = initialState, action) {
```

Comment:
I confused myself for a second and then felt compelled to write it down once I came back to reality.